src/node.js

```diff
@@ -178,12 +178,29 @@
   }
 
   startup.setupProcessObject = function() {
-    process._setupProcessObject(setPropByIndex);
+    const _hrtime = process.hrtime;
+    var hrValues = new Uint32Array(4);
```

Comment:
Future optimization: instead of allocating many small but long-lived typed arrays in many places, allocate a single one early on and use fixed offsets in places like this.